eval/eval.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+
+
+set -e
+set -o pipefail
+
+mkdir -p $EVAL_DIR
+mkdir -p $EVAL_DIR/$SRC-$TRG
+
+sacrebleu -t $DATASET -l $SRC-$TRG --echo src \
+    | tee $EVAL_DIR/$SRC-$TRG/$DATASET.$SRC \
+    | $TRANSLATOR_CMD \
+    | tee $EVAL_DIR/$SRC-$TRG/$DATASET.$TRANSLATOR.$TRG \
+    | sacrebleu --score-only -q -t $DATASET -l $SRC-$TRG \
+    | tee $EVAL_DIR/$SRC-$TRG/$DATASET.$TRANSLATOR.$TRG.bleu
```

Comment:
Nit: Ending files with newline makes diffs easier to read after file is appended in future revisions